### PR TITLE
Add new version of LightTextClassification

### PR DESCRIPTION
**Pull request by andznui**
- zip uploaded in publicmodels and in download containers (https://uipath.blob.core.windows.net/download/AIC/LightTextClassification/23.10.3.0/bow_text_classifier_package.zip)

### CODE REVIEW

File: metadata/LightTextClassification__5__metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "mlPackageLanguage": "PYTHON38",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Language Analysis",
+    "projectDescription": "Curated Models from UiPath to analyze language in Emails, Documents, Web Pages, Text Messages, Portals and more",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v22.4",
+    "version": 4,
+    "customVersion": "23.10.3.0",
+    "contentUri": "https://<placeholder-url>/publicmodels/AIC/LightTextClassification/4/bow_text_classifier_package.zip"
```
Comment (andznui) [inline metadata/LightTextClassification__5__metadata.json]:
```suggestion
    "contentUri": "https://<placeholder-url>/publicmodels/AIC/LightTextClassification/23.10.3.0/bow_text_classifier_package.zip"
```

File: metadata/LightTextClassification__5__metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON38",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Language Analysis",
+    "projectDescription": "Curated Models from UiPath to analyze language in Emails, Documents, Web Pages, Text Messages, Portals and more",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v22.4",
+    "version": 4,
+    "customVersion": "23.10.3.0",
```
Comment (irinel-birsan-uipath) [inline metadata/LightTextClassification__5__metadata.json]: let's keep it 23.10.3

File: metadata/LightTextClassification__5__metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "contentUri": "https://<placeholder-url>/publicmodels/AIC/LightTextClassification/23.10.3.0/bow_text_classifier_package.zip"
```
Comment (irinel-birsan-uipath) [inline metadata/LightTextClassification__5__metadata.json]: And here change it to `23.10.3.0` in path

Comment (andznui) [inline metadata/LightTextClassification__5__metadata.json]:
```suggestion
    "contentUri": "https://<placeholder-url>/publicmodels/AIC/LightTextClassification/23.10.3/bow_text_classifier_package.zip"
```

Comment (irinel-birsan-uipath) [reply metadata/LightTextClassification__5__metadata.json]: + make sure the file exists

File: metadata/LightTextClassification__5__metadata.json (hunk 2 above)
Comment (andznui) [inline metadata/LightTextClassification__5__metadata.json]:
```suggestion
    "customVersion": "23.10.3",
```

File: metadata/LightTextClassification__5__metadata.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    "processorType": "CPU",
+    "description": "This is the preview version of a generic, retrainable model for text classification. It supports all languages based on Latin characters, such as English, French, Spanish, and others. This ML Package must be trained, and if deployed without training first, the deployment will fail with an error stating that the model is not trained. This model operates on bag of Words and supports two variations of algorithms, one based on Logistic Regression and the other one based on Random Forest. You can switch between these two algorithms using environment variables.",
+    "inputDescription": "Text that would be classified.",
+    "outputDescription": "A JSON with predicted class name, associated confidence on that class prediction (between 0-1) and ngrams in the input that affected the prediction (optional)",
+    "changeLog": "model 22.6.0",
+    "cpu": 1,
+    "inputType": "JSON",
+    "displayName": "LightTextClassification",
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON38",
```
Comment (andznui) [inline metadata/LightTextClassification__5__metadata.json]:
```suggestion
    "mlPackageLanguage": "PYTHON39",
```

**Review by sstefanov78 — APPROVED** [review]

**Review by florin-szilagyi — APPROVED** [review]

**Pull request merged**